src/utils/data_transform.py

```diff
@@ -39,7 +41,9 @@ def __init__(
         self.temporal_augmentation = temporal_augmentation
 
         if temporal_augmentation:
-            self.temporal_augmentation_model = torch.nn.Identity()
+            self.temporal_augmentation_model = LSTM(
+                128, 0.57, True  
+            )
```

Comment:
I don't think this line is going to work, did you try and run it?

Reply:
> You can "run" it, but it's not done yet so it will crash. I just put this incomplete one since you needed my basic implementation to add your part in.

Reply:
The work is still in progress, so please gimme some more time.

Reply:
Ok, I think you can leave this line as it was. We can train separately for now